lib/emoji.rb

```diff
@@ -24,14 +24,23 @@ def all
     @all
   end
 
-  def apple_palette
-    return @apple_palette if defined? @apple_palette
-    data = File.open(apple_palette_file, 'r:UTF-8') { |f| JSON.parse(f.read) }
-    @apple_palette = data.fetch('EmojiDataArray').each_with_object({}) do |group, all|
-      title = group.fetch('CVDataTitle').split('-', 2)[1]
-      all[title] = group.fetch('CVCategoryData').fetch('Data').split(',').map do |raw|
-        TEXT_GLYPHS.include?(raw) ? raw + VARIATION_SELECTOR_16 : raw
+  def palette
```

Comment:
Since people are going to use this method much like they used `apple_palette`, let's also alias this method as `apple_palette` for backwards compatibility.

Reply:
Sure! I was mostly thinking about GitHub's use-case here and I couldn't find any references to this method in `github/github`. I'll alias this.